### PR TITLE
Oc 293 upgrade flyway version

### PR DESCRIPTION
**Pull request by robindenadel**

### CODE REVIEW

File: osgp-domain-core/pom.xml [hunk 1]
```diff
@@ -135,7 +135,7 @@
 
         <!-- Flyway (DB Migration) -->
     <dependency>
-      <groupId>com.googlecode.flyway</groupId>
+      <groupId>org.flywaydb</groupId>
```
Comment (smvdheijden) [inline osgp-domain-core/pom.xml]: Remove unused dependency

File: osgp-adapter-ws-admin/pom.xml [hunk 2]
```diff
@@ -158,7 +158,7 @@
 
     <!-- Flyway (DB Migration) -->
     <dependency>
-      <groupId>com.googlecode.flyway</groupId>
+      <groupId>org.flywaydb</groupId>
```
Comment (smvdheijden) [inline osgp-adapter-ws-admin/pom.xml]: Remove unused dependency

File: osgp-adapter-ws-core/pom.xml [hunk 3]
```diff
@@ -161,7 +161,7 @@
 
     <!-- Flyway (DB Migration) -->
     <dependency>
-      <groupId>com.googlecode.flyway</groupId>
+      <groupId>org.flywaydb</groupId>
```
Comment (smvdheijden) [inline osgp-adapter-ws-core/pom.xml]: Remove unused dependency

File: osgp-adapter-ws-publiclighting/pom.xml [hunk 4]
```diff
@@ -145,7 +145,7 @@
 
     <!-- Flyway (DB Migration) -->
     <dependency>
-      <groupId>com.googlecode.flyway</groupId>
+      <groupId>org.flywaydb</groupId>
```
Comment (smvdheijden) [inline osgp-adapter-ws-publiclighting/pom.xml]: Remove unused dependency

File: osgp-adapter-ws-tariffswitching/pom.xml [hunk 5]
```diff
@@ -145,7 +145,7 @@
 
     <!-- Flyway (DB Migration) -->
     <dependency>
-      <groupId>com.googlecode.flyway</groupId>
+      <groupId>org.flywaydb</groupId>
```
Comment (smvdheijden) [inline osgp-adapter-ws-tariffswitching/pom.xml]: Remove unused dependency

File: osgp-domain-core/pom.xml (hunk 1 above)
Comment (robindenadel) [inline osgp-domain-core/pom.xml]: ok

File: osgp-adapter-ws-admin/pom.xml (hunk 2 above)
Comment (robindenadel) [inline osgp-adapter-ws-admin/pom.xml]: ok

File: osgp-adapter-ws-core/pom.xml (hunk 3 above)
Comment (robindenadel) [inline osgp-adapter-ws-core/pom.xml]: ok

File: osgp-adapter-ws-publiclighting/pom.xml (hunk 4 above)
Comment (robindenadel) [inline osgp-adapter-ws-publiclighting/pom.xml]: ok

File: osgp-adapter-ws-tariffswitching/pom.xml (hunk 5 above)
Comment (robindenadel) [inline osgp-adapter-ws-tariffswitching/pom.xml]: ok

**Pull request merged**